backend/backend.go

```diff
@@ -683,3 +685,62 @@ func (backend *Backend) HandleURI(uri string) {
 		backend.log.Warningf("Unknown URI scheme: %s", uri)
 	}
 }
+
+// PocketWidgetSignAddress returns an unused address and makes the user sign a message to prove ownership.
+func (backend *Backend) PocketWidgetSignAddress(code accounts.Code, msg string, format string) (string, string) {
+	log := backend.log.WithField("accountCode", code)
+	var account accounts.Interface
+	for _, acct := range backend.accounts {
```

Comment:
move `getAccountFromCode()` from the handlers here and use that in both places. That one also checks if the account is active.

Reply:
Done :+1: